### ISSUE TITLE
Hello, world!

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-jobqueue#1]
@jhamman @jedwards4b I've moved the job-queuing contents of the pangeo repository here.  Some questions:

1.  Are you comfortable losing commit history.  I've pointed to the origin commit in the first commit of this repo.  I've also mentioned you both in the history section of the README (which would eventually go to a docpage?)
2.  How would you like to license this?  I'm inclined to use the same license file used in Dask: https://github.com/dask/dask/blob/master/LICENSE.txt

@jakirkham (and anyone else actively using dask-drmaa) you should probably know about this parallel effort.  I may start thinking about pulling a little bit of the shared logic between these systems into dask.distributed.

also cc @rabernat

**Comment by mrocklin** [comment]
Oh, and if anyone has a better name, that would be welcome.  I've copied over what was in the pangeo repository.

**Comment by jakirkham** [comment]
Thanks @mrocklin for doing this and everyone involved in writing and sharing the code. :)

Had been thinking recently that it would be nice if there was something to construct shim shell scripts for getting `dask-worker` jobs launched on schedulers and connected back to a Distributed Client. Was about to raise an issue over at Distributed where it may or may not have been appropriate. Happy to see this emerge around the same time. :)

**Comment by jhamman** [comment]
I don't have any problem using the dask license or loosing my commit history, remarkable as it was.

**Comment by guillaumeeb** [comment]
Hi everyone,

I 've followed Dask and Pangeo for a few months now, and I was wondering where to start to collaborate with you guys. I feel that this may be the right place for me, as it is the closer to my work environment (Dask on PBS cluster). 

I will follow the issues opened here, but do not hesitate to cc me if you need some test on an alternative environment, and tell me if you think about a simple task on which you need some manpower.

Cheers!

**Comment by mrocklin** [comment]
Thanks @guillaumeeb !  I see three main directions of work short term:

1.  Build CI systems.  This requires some understanding of docker and docker-compose.  See #2 
2.  Add new implementations for new job schedulers.  Given that you are most familiar with PBS and that PBS already exists this might not be a good fit.
3.  Proceed through the actions of a beginning user and try to use PBSCluster on your cluster.  Report back any confusion or bugs that you encounter.  This code was developed on a couple of PBS instances, but I suspect that it is not yet completely general and will fail in interesting ways on other PBS deployments.

**Comment by jedwards4b** [comment]
I don't have any concerns with license or history - thanks for taking this.

**Comment by jakirkham** [comment]
To finish up item 2, have put up PR ( https://github.com/dask/dask-jobqueue/pull/9 ), which just copies Dask's License file and sets the copyright year to 2018. Sounds like we are on agreement on that. Also includes some stuff to package the license and some misc packaging things. Requested reviews of the original code owners. Hope that is ok. :)

**Comment by guillaumeeb** [comment]
@mrocklin 
I can begin with point 3. Do you foresee to add some beginner documentation on how to setup the module, or even more, on how to set up a dask python environment with job queue, to have the prerequisite to run the example in the Readme (I could write it, or can we point to some equivalent documentation)? Should we add some cluster scheduler specific documentation on how to check that the Dask cluster has started (eg. using qstat for PBS)?
The point 3 will give me some insights to help on issue #7 also.

I will then be happy to help on point 1, so issue #2, by beginning to set up a PBS docker environment, if no one else has the bandwith to do it! Hope I can find the time.

**Comment by mrocklin** [comment]
I would install dask: http://dask.pydata.org/en/latest/install.html

I would then pip install from this repository:

    pip install git+https://github.com/dask/dask-jobqueue

We don't have anything on PyPI or conda-forge yet.  Having basic documentation on RTD would be nice in the future.

> Should we add some cluster scheduler specific documentation on how to check that the Dask cluster has started

Sure?

**Comment by guillaumeeb** [comment]
Okay, had never installed a Python module from git before, no idea it was so simple ^^'. 
For the documentation, shall we initialize a docs folder with basic files for Sphinx and Read The Docs integration?

Sorry for the basic questions I am (and will be) asking, beginner on Open source Development here.

**Comment by guillaumeeb** [comment]
So here are my first report on trying PBSCluster on our environment:
 * It works 👍, an it is really easy to use. Really good work, thanks!
 * I am concerned by the fact that there is no options for configuring the `--local-directory` of the dask-worker. We could use the `extra` configuration option, but I would prefer to have a named option, which is linked to #7 reflexion. In particular, our PBS conf sets a $TMPDIR env var that should be used.
 * I could not find a simple `cluster.stop()` method. I tried to use `cluster.scheduler.close()`, and take advantage of the death-timeout, but apparently it did not work. I think it would be good to have such `cluster.stop()` method that stops every PBS jobs and the sheduler process.
 * One minor point (and that's probably because I did not search long enough), I could not find a method to get the info about both scheduler and bokeh bindings ip:port, like what can be found in a scheduler.json written file.
